client/src/components/App/Issue.css

```diff
@@ -2,3 +2,15 @@
   display: flex;
   flex-grow: 4;
 }
+
+.infoTags {
+  position: flex;
+  float: right;
+
+  font-size: 0.7em;
+  background-color: hotpink;
```

Comment:
Excellent choice of color, but maybe try something else?

Maybe the same color which is used in the current vote status bars on the right? `#e0e2ef`